api/v1alpha1/inferencemodel_types.go

```diff
@@ -90,47 +101,34 @@ type TargetModel struct {
 	Weight int `json:"weight,omitempty"`
```

Comment:
Can't comment on lines above this, but recommend updating the spec here to be more descriptive. Weight isn't necessarily a percentage. Recommend looking at [corresponding Gateway API spec here](https://github.com/kubernetes-sigs/gateway-api/blob/27b4aa839015274a113ee355ec22210e816e7a40/apis/v1/shared_types.go#L268-L278) as a reference.

Reply:
Agreed, the implementation doesnt expect a % either:  https://github.com/kubernetes-sigs/llm-instance-gateway/blob/918960cf7f4b2d53d912e6a08b7378f20d20bef7/pkg/ext-proc/backend/datastore.go#L86
Will change the word to `proportion`